### PR TITLE
Add Catalan support and translations

### LINKED ISSUE #1426: 🚀 Feature: i18n - Add catalan support
### 🔖 Feature description

Adding Catalan language support

### 🎤 Why is this feature needed ?

To reach more customers with a personalized UI for their language

### ✌️ How do you aim to achieve this?

I will add a new translations file in `/packages/notification-center/src/i18n/languages/ca.ts`.
Import the translations in `lang.ts` file, in the parent directory.
Add the new language to docs.
Submit a PR.

### 🔄️ Additional Information

_No response_

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by pmontp19**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR adds Catalan language support

- **Why this change was needed?** (You can also link to an open issue here)
This fixes #1426

### CODE REVIEW

File: packages/notification-center/src/i18n/languages/ca.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const CA: ITranslationEntry = {
+  translations: {
+    notifications: 'Notificacions',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/ca.ts:5]: Could you add the Settings translation?
```suggestion
    settings: 'Configuració',
```

**Review by p-fernandez — COMMENTED** [review]
Gràcies. 🙂

File: packages/notification-center/src/i18n/languages/ca.ts (hunk 1 above)
Comment (pmontp19) [reply packages/notification-center/src/i18n/languages/ca.ts:5]: done!

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**